CHANGELOG.md

```diff
@@ -22,6 +26,10 @@ You may also find the [Update Guide](UPDATING.md) useful.
 - Linux, NetBSD, Solaris: read in blocking mode on first use in `fill_bytes`. (#484)
 - Fuchsia, CloudABI: fix compilation (broken in Rand 0.5). (#484)
 
+### Other changes
+- References to a `CryptoRng` now also implement `CryptoRng`. (#470)
+- Fix example code in README. (#482)
```

Comment:
Really worth mentioning?

Reply:
Maybe not, but not really worth removing either.